guides/release/configuring-ember/debugging.md

```diff
@@ -70,15 +70,15 @@ export default Application.extend({
   LOG_RESOLVER: true
 });
 ```
-#### Dealing with deprecations
+### Dealing with deprecations
```

Comment:
We should talk about ember-cli-deprecation-workflow  it these flags

Reply:
https://github.com/ember-learn/guides-source/issues/475